src/frontend/js/components/Button/index.tsx

```diff
@@ -0,0 +1,9 @@
+interface ButtonProps {
+  label: string;
+  color: 'primary' | 'secondary' | 'outline-primary' | 'outline-secondary';
+}
+
+export const Button = (props: ButtonProps) => {
+  const classes = ['button', 'button--' + props.color];
+  return <button className={classes.join(' ')}>{props.label}</button>;
```

Comment:
I'll use `children` instead of a `label` property.
```suggestion
  return <button className={classes.join(' ')}>{children}</button>;
```